### PR TITLE
German Corne Layout

### PR DESCRIPTION
**Pull request by JulianTurner**
I added a german Layout with a symbol & navigation layer based on NEOQWERTZ

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/julian_turner/oled.c [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "layers.h"
+
+char wpm_str[10];
```
Comment (drashna) [inline keyboards/crkbd/keymaps/julian_turner/oled.c:19]: This shouldn't be needed anymore. 
```suggestion
```

**Review by drashna — APPROVED** [review]

File: keyboards/crkbd/keymaps/julian_turner/rules.mk [hunk 2]
```diff
@@ -0,0 +1,15 @@
+RGBLIGHT_ENABLE = yes    # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/julian_turner/rules.mk:1]:
```suggestion
RGBLIGHT_ENABLE = yes
```

File: keyboards/crkbd/keymaps/julian_turner/keymap.c [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "keycodes.h"
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/julian_turner/keymap.c:18]:
```suggestion
#include "keymap_german.h"
#include "layers.h"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by JulianTurner — COMMENTED** [review]
implementation of suggestions

**Review by drashna — APPROVED** [review]

**Comment by JulianTurner** [comment]
Is there anything required on my end?

File: keyboards/crkbd/keymaps/julian_turner/readme.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# JulianTurner´s CRKBD Layout
+
+## Compile
+
+Compile using `qmk compile -kb crkbd -km JulianTurner` for Pro Micro
```
Comment (noroadsleft) [inline keyboards/crkbd/keymaps/julian_turner/readme.md:5]:
```suggestion
Compile using `qmk compile -kb crkbd -km julian_turner` for Pro Micro
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**